assets/js/components/Link.js

```diff
@@ -35,13 +40,26 @@ function Link( {
 	caps,
 	danger,
 	disabled,
+	'aria-label': ariaLabel,
 	...extraProps
 } ) {
 	// Note: the disabled attribute does not alter behavior of anchor tags,
 	// so if disabled we force it to be a button.
 	const isAnchor = href && ! disabled;
 	const SemanticLink = isAnchor ? 'a' : 'button';
 
+	const getAriaLabel = () => {
+		let label = ariaLabel;
+
+		if ( ! external ) {
+			return label;
+		} else if ( typeof children === 'string' ) {
```

Comment:
(It seems like it might be fine, I'm just not sure 🙂 )

Reply:
> But in this case, if `children` is a string it will override the ARIA label, even when one was provided.

That's not the case, because the very next line has `label || children` 😄 

I suppose we could change the conditional but I thought it was actually easier to read like that but apparently not!

`children` can be a string (or a function or anything else), so this works as expected in my testing. I'll add a few tests here that I wrote to check this so we don't need to worry about it going forward 🙂